### PR TITLE
ported process-list

### PR DESCRIPTION
**Pull request by galczo5**

### CODE REVIEW

**Comment by shaleh** [comment]
@galczo5 it is just a minor formatting nit. You have a space between a variable and a parens.

**Comment by galczo5** [comment]
Done, travis still has "failed" flag.

**Comment by brotzeit** [comment]
Try running rustfmt.

**Comment by galczo5** [comment]
Done

File: rust_src/src/process.rs [hunk 1]
```diff
@@ -70,3 +70,8 @@ pub fn get_buffer_process(buffer: LispObject) -> LispObject {
     }
     return LispObject::constant_nil();
 }
+
+#[lisp_fn]
+pub fn process_list() -> LispObject {
```
Comment (brotzeit) [inline rust_src/src/process.rs]: You forgot to add the docstring.

**Comment by brotzeit** [comment]
Thanks! BTW you can [close issues automatically](https://help.github.com/articles/closing-issues-using-keywords/).

**Pull request merged**